daml-script/export/integration-tests/reproduces-transactions/test/scala/com/daml/script/export/ReproducesTransactions.scala

```diff
@@ -307,12 +295,16 @@ trait ReproducesTransactions
       } yield ()
   }
 
-  def numParties: Int
-  def skip: Int
-  def description: String
+  "Generated export for IOU transfer compiles" - {
```

Comment:
any reason why you don't have three tests below this rather than having three separate top-level blocks?